public/less/_admin.less

```diff
@@ -64,4 +64,9 @@
             margin-left: 5px;
         }
     }
+
+    code, pre {
```

Comment:
`pre` is only used why add also `code`?
```suggestion
    pre {
```

Reply:
`<code>` is also used on admin page and this avoids the issue in other fields (e.g. mobile) or move all formatted stuff to pre

Reply:
wouldn't just styling `code` fix also this without changing tags from `code` to `pre`?

Reply:
thats why i'm added code aswell

Reply:
All lines with `JsonPrettyPrint` in config use `<pre>`. I want a consistent behavior/style on same page.